pom.xml

```diff
@@ -2,13 +2,19 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.neo4j</groupId>
     <artifactId>neo4j-jdbc</artifactId>
-    <version>2.3-SNAPSHOT</version>
+    <version>2.0.5-SNAPSHOT</version>
```

Comment:
Should the version of the artifact change as well, or is it just the result of an over-eager match?